src/domains/idpay-app/terraform.sh

```diff
@@ -1,46 +1,192 @@
 #!/bin/bash
+# Terraform script for managing infrastructure on Azure
```

Comment:
Last version of this script is v1.3
I suggest you to use a symbolic link. The original script(-s) is/are located into the following folder:
<git repo root>/scripts/

Reply:
done